content/en/docs/gke/pipelines/upgrade.md

```diff
@@ -1,134 +1,55 @@
 
-* You must use command-line deployment. 
-  You cannot reinstall Kubeflow Pipelines using the web interface.
-* When you do `kfctl apply` or `kfctl build`, you should use a different 
-  deployment name from your existing deployment name. Otherwise, kfctl will
-  delete your data in the existing PDs. This guide defines the deployment name
-  in the ${KF_NAME} environment variable. 
+## Kubeflow Pipelines Standalone
 
-To reinstall Kubeflow Pipelines:
+Upgrade Support for Kubeflow Pipelines standalone is **Beta**.
 
-1. Follow the [command-line deployment 
-  instructions](/docs/gke/deploy/deploy-cli/), but **note the following
-  changes in the procedure**.
+[Upgrading Kubeflow Pipelines Standalone](/docs/pipelines/installation/standalone-deployment/#upgrading-kubeflow-pipelines) introduces how to upgrade in place.
 
-1. Set a different `${KF_NAME}` name from your existing `${KF_NAME}`.
+## Full Kubeflow
 
-1. **Before** running the `kfctl apply` command:
+On Google Cloud, the full Kubeflow follows [the blueprint pattern](https://googlecontainertools.github.io/kpt/guides/producer/blueprint/) starting from Kubeflow 1.1.
 
-    * Edit `${KF_DIR}/gcp_config/storage-kubeflow.yaml` and set the following
-      flag to skip creating new storage:
+The blueprint pattern makes it easy to upgrade the full Kubeflow in-place, refer to [Update Kubeflow on Google Cloud](docs/gke/deploy/deploy-cli/#update-kubeflow) documentation for instructions.
 
-      ```
-      ...
-      createPipelinePersistentStorage: false
-      ...
-      ```
+However, there's no current support to upgrade from Kubeflow 1.0 or earlier to Kubeflow 1.1 whiling keeping Kubeflow Pipelines data. Provide your feedback in [kubeflow/pipelines#4346](https://github.com/kubeflow/pipelines/issues/4346) if it's important to you.
 
-    * Edit `${KF_DIR}/kustomize/minio/overlays/minioPd/params.env` and specify 
-      the PD that your existing deployment uses for the Minio server:
+## AI Platform Pipelines
 
-        ```
-        ...
-        minioPd=[NAME-OF-ARTIFACT-STORAGE-DISK]
-        ...
-        ```
+Upgrade Support for AI Platform Pipelines is **Alpha**.
 
-    * Edit `${KF_DIR}/kustomize/mysql/overlays/mysqlPd/params.env` and specify 
-      the PD that your existing deployment uses for the MySQL database:
+To upgrade your AI Platform Pipelines instance while keeping existing data:
 
-        ```
-        ...
-        mysqlPd=[NAME-OF-METADATA-STORAGE-DISK]
-        ...
-        ```
+### For instances **without** managed storage:
 
-1. Run the `kfctl apply` command to deploy Kubeflow as usual:
+1. [Delete your AI Platform Pipelines instance](https://cloud.google.com/ai-platform/pipelines/docs/getting-started#clean_up) **without** selecting **Delete cluster**. The persisted artifacts and database data are stored in persistent volumes in the cluster. They are kept by default when you do not delete the cluster.
```

Comment:
Suggestion:

- _italicize_ "without" without **boldening**, as the latter is useful for names of buttons or options

```diff
- ... **without** selecting **Delete cluster**.
+ ... _without_ selecting **Delete cluster**.
```

Reply:
I used boldening because it's very important to remember `without`, if user accidentally chose `Delete cluster`, they will lose all their data.